client/my-sites/theme/main.jsx

```diff
@@ -577,6 +595,13 @@ const ThemeSheetWithOptions = ( props ) => {
 	);
 };
 
+const mapDispatchToProps = ( dispatch, ownProps ) => ( {
```

Comment:
Plus 1 for consistency, though `mapStateToProps` is long enough here that we should probably pull that out into it's own function.